### PR TITLE
Complete about me page

### PR DESCRIPTION
**Pull request by binaruappuhamy**
Verify completion of about me page

### CODE REVIEW

File: src/HomePage.js [hunk 1]
```diff
@@ -5,8 +5,18 @@ class HomePage extends React.Component {
   render() {
     return (
       <div>
-        {/* Complete your training in here*/}
-        <div>Welcome to VEEP Website!</div>
+        <div align='center'>
+          <img src="uoft_logo.jpg" alt="UofT Logo" width="179" height="79" />
```
Comment (CatherineZM) [inline src/HomePage.js:9]: In the future, we will keep the styling in CSS just keep in mind

**Review by CatherineZM — APPROVED** [review]

**Review by CatherineZM — APPROVED** [review]
In the future, we will keep styling in the CSS files just keep in mind.

**Comment by CatherineZM** [comment]
@Felix-Deng @iiiizuo can you also take a look when you have a chance?

**Review by Felix-Deng — APPROVED** [review]
Well done, Binaru!

**Pull request closed without merging**